com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -53,12 +53,21 @@ internal struct AgentInfo
 
         public void ClearActions()
         {
-            Array.Clear(storedVectorActions, 0, storedVectorActions.Length);
+            storedVectorActions.Clear();
         }
 
         public void CopyActions(ActionBuffers actionBuffers)
         {
-            actionBuffers.PackActions(storedVectorActions);
```

Comment:
Do we use PackActions anywhere else? If not, can we remove it?

Reply:
removed